source/blood/src/gamemenu.cpp

```diff
@@ -122,6 +124,16 @@ bool CGameMenuMgr::Push(CGameMenu *pMenu, int nItem)
     return true;
 }
 
+void CGameMenuMgr::MoveMouseToCenter()
+{
```

Comment:
I don't think using SDL directly here is a good idea, either move this to the engine or make this code active only for SDL builds